src/commands/info/releasenotes/display.ts

```diff
@@ -90,16 +92,33 @@ export default class Display extends SfdxCommand {
         const footer = messages.getMessage('footer', [this.config.bin, releaseNotesPath, HIDE_NOTES, HIDE_FOOTER]);
 
         this.ux.log(marked.parse(footer));
+      } else {
+        // footer hidden
+        await Lifecycle.getInstance().emitTelemetry({ eventName: 'FOOTER_HIDDEN' });
```

Comment:
Hey @WillieRuemmele, I think I did a bad job explaining this one. The metric I'm wanting to collect here is how many people are actively setting the hide footer env var. This `else` will emit when someone runs a command on their machine (not with the `--hook` flag that will run in the post update hooks). Will want to make sure `isHook` and the env var are both true, something like:

```js
if (isHook) {
  if (env.getBoolean(HIDE_FOOTER)) {
    await Lifecycle.getInstance().emitTelemetry({ eventName: 'FOOTER_HIDDEN' });
  } else {
    const footer = messages.getMessage('footer', [this.config.bin, releaseNotesPath, HIDE_NOTES, HIDE_FOOTER]);

    this.ux.log(marked.parse(footer));
  }
}
```